<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaffirePro14Monitoring</class>
 <widget class="QWidget" name="SaffirePro14Monitoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saffire Pro 40 Monitoring</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="GlobalSetMain">
     <item>
      <widget class="QGroupBox" name="GlobalSet">
       <property name="whatsThis">
        <string comment="Activate/Deactivate the global volume diminution" extracomment="Activate/Deactivate the global volume diminution"/>
       </property>
       <property name="title">
        <string>Global Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QPushButton" name="HWSwitch">
          <property name="text">
           <string>H/W</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="GlobalSetLabel">
          <property name="text">
           <string>Dim level</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QSlider" name="DimLevel">
            <property name="minimum">
             <number>-87</number>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="GlobalDim">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Activate Dim</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="GlobalMute">
          <property name="text">
           <string>Mute</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="LineOutMain">
     <item>
      <widget class="QGroupBox" name="LineOut">
       <property name="title">
        <string>Line/Out Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="MonoMain">
          <item>
           <widget class="QGroupBox" name="Mono">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="Mono_12">
               <property name="text">
                <string>Mono</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="Mono_34">
               <property name="text">
                <string>Mono</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="LineOutMain_1">
            <item>
             <widget class="QGroupBox" name="LineOut_1">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_1">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QSlider" name="LineVolume_1">
                   <property name="locale">
                    <locale language="French" country="France"/>
                   </property>
                   <property name="minimum">
                    <number>-40</number>
                   </property>
                   <property name="maximum">
                    <number>0</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBothSides</enum>
                   </property>
                   <property name="tickInterval">
                    <number>5</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="LineUnActivate_1">
                 <property name="text">
                  <string>UA</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineMute_1">
                 <property name="text">
                  <string>Mu</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineGDim_1">
                 <property name="text">
                  <string>GD</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineGMute_1">
                 <property name="text">
                  <string>GM</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="LineOutMain_2">
            <item>
             <widget class="QGroupBox" name="LineOut_2">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>2</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QSlider" name="LineVolume_2">
                   <property name="minimum">
                    <number>-40</number>
                   </property>
                   <property name="maximum">
                    <number>0</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBothSides</enum>
                   </property>
                   <property name="tickInterval">
                    <number>5</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="LineUnActivate_2">
                 <property name="text">
                  <string>UA</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineMute_2">
                 <property name="text">
                  <string>Mu</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineGDim_2">
                 <property name="text">
                  <string>GD</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineGMute_2">
                 <property name="text">
                  <string>GM</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="LineOutMain_3">
            <item>
             <widget class="QGroupBox" name="LineOut_3">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>3</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QSlider" name="LineVolume_3">
                   <property name="minimum">
                    <number>-40</number>
                   </property>
                   <property name="maximum">
                    <number>0</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBothSides</enum>
                   </property>
                   <property name="tickInterval">
                    <number>5</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="LineUnActivate_3">
                 <property name="text">
                  <string>UA</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineMute_3">
                 <property name="text">
                  <string>Mu</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineGDim_3">
                 <property name="text">
                  <string>GD</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineGMute_3">
                 <property name="text">
                  <string>GM</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="LineOutMain_4">
            <item>
             <widget class="QGroupBox" name="LineOut_4">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>4</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QSlider" name="LineVolume_4">
                   <property name="minimum">
                    <number>-40</number>
                   </property>
                   <property name="maximum">
                    <number>0</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBothSides</enum>
                   </property>
                   <property name="tickInterval">
                    <number>5</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="LineUnActivate_4">
                 <property name="text">
                  <string>UA</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineMute_4">
                 <property name="text">
                  <string>Mu</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineGDim_4">
                 <property name="text">
                  <string>GD</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="LineGMute_4">
                 <property name="text">
                  <string>GM</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QGridLayout" name="LineInSwitchesMain">
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="LineInSwitches">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Line/In switches</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QGroupBox" name="LineInSwitch_1">
            <property name="title">
             <string>1</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="LineInSwitchLine_1">
               <property name="text">
                <string>Line</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="LineInSwitchInst_1">
               <property name="text">
                <string>Inst</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="LineInSwitch_2">
            <property name="title">
             <string>2</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="LineInSwitchLine_2">
               <property name="text">
                <string>Line</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="LineInSwitchInst_2">
               <property name="text">
                <string>Inst</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="LineInSwitch_3">
            <property name="title">
             <string>3</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="LineInSwitchLo_3">
               <property name="text">
                <string>Lo</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_3</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="LineInSwitchHi_3">
               <property name="text">
                <string>Hi</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_3</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="LineInSwitch_4">
            <property name="title">
             <string>4</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="LineInSwitchLo_4">
               <property name="text">
                <string>Lo</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_4</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="LineInSwitchHi_4">
               <property name="text">
                <string>Hi</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_4</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
